swarm/storage/mru/resourceviewid.go

```diff
@@ -1,9 +1,7 @@
 package mru
```

Comment:
just call this file `view`

Reply:
I know, I didn't want to rename files until the very end to avoid large diffs that don't tell you what changed. You'd see a deleted file and a new file.